easybuild/easyconfigs/p/Pillow-SIMD/Pillow-SIMD-6.0.x.post0-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'Pillow-SIMD'
+version = '6.0.x.post0'
+
+homepage = 'https://github.com/uploadcare/pillow-simd'
+description = """Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.
+ PIL is the Python Imaging Library by Fredrik Lundh and Contributors."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = ['https://github.com/uploadcare/pillow-simd/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['7bccd59fa8b47f63591a993691de550a21e7ace4b92b3cd38dad9472b01fb008']
+
+builddependencies = [('binutils', '2.31.1')]
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+dependencies = [
+    ('libjpeg-turbo', '2.0.2'),
+    ('libpng', '1.6.36'),
+    ('zlib', '1.2.11'),
+    ('LibTIFF', '4.0.10'),
+    ('freetype', '2.9.1')
+]
+
+use_pip = True
+download_dep_fail = True
+
+options = {'modulename': 'PIL'}
+
```

Comment:
As with the other PR, I think it's worth having a `sanity_check`, even when using `pip`.

Reply:
@verdurin PythonPackage easyblocks already have a default `sanity_check_paths` and `sanity_check_commands`, do you want me to add a custom one?